subsys/mgmt/mcumgr/Kconfig

```diff
@@ -155,6 +391,4 @@ config MCUMGR_BUF_USER_DATA_SIZE
 
 	  Note that CONFIG_NET_BUF_USER_DATA_SIZE must be at least as big as
 	  MCUMGR_BUF_USER_DATA_SIZE.
-
```

Comment:
I think it is more common in Kconfig files to always have one NL before and one NL after `if` and `endif` statements.

Reply:
I think this varies to be honest.